Production/crab/configs/DeepTau2023/Winter22Test.txt

```diff
@@ -0,0 +1,6 @@
+/DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Winter22DRPremix-ALCARECO_122X_mcRun3_2021_realistic_v9_ext1-v3/GEN-SIM-RAW
+/DYToLL_M-50_TuneCP5_13p6TeV-pythia8/Run3Winter22DR-L1TPU0to99FEVT_122X_mcRun3_2021_realistic_v9-v2/GEN-SIM-DIGI-RAW
+/VBFHToTauTau_M125_TuneCP5_13p6TeV-powheg-pythia8/Run3Winter22DR-L1TPU0to99FEVT_122X_mcRun3_2021_realistic_v9-v2/GEN-SIM-DIGI-RAW
+/QCD_Pt-15to7000_TuneCP5_Flat_13p6TeV-pythia8/Run3Winter22DR-PUForTRK_DIGI_122X_mcRun3_2021_realistic_v9-v2/GEN-SIM-DIGI-RAW
+/TT_TuneCP5_13p6TeV-powheg-pythia8/Run3Winter22DR-PUForTRK_DIGI_122X_mcRun3_2021_realistic_v9-v2/GEN-SIM-DIGI-RAW
+/WJetsToLNu_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Winter22DRPremix-TRKALCADesign_design_geometry_122X_mcRun3_2021_design_v9-v2/GEN-SIM-RAW
```

Comment:
I propose to organize this list a bit differently:
1) As it is HLT related datasets, and format is different from miniAOD, we could name folders with HLT_ prefix. Also by convention we use MC period, So instead of DeepTau2023 I propose HLT_Run3Winter22
2) Can this config be organized properly according to what is needed to be provided to the crab submitting script?

Reply:
Hi, 
thanks a lot for the suggestion, for 1: have changed to the proposed name. For 2, I am not sure what is the correct structure for crab submitting, should I follow the ones in Run3Winter22?

Reply:
Hi,
thanks a lot for the suggestion, for 1: have changed to the proposed name. For 2, I am not sure what is the correct structure for crab submitting, should I follow the ones in Run3Winter22?

Reply:
Yes, something like configs from `Production/crab/configs/*UL` or `Run3Winter22` folders, to make sure it is parsed correctly by `crab_submit.py`